packages/compiler/src/server/lib-provider.ts

```diff
@@ -1,54 +1,57 @@
 import { joinPaths } from "../core/path-utils.js";
 import { NpmPackage, NpmPackageProvider } from "./npm-package-provider.js";
 
-export class EmitterProvider {
-  private isEmitterPackageCache = new Map<string, boolean>();
-  constructor(private npmPackageProvider: NpmPackageProvider) {}
+export class LibraryProvider {
+  private isLibPackageCache = new Map<string, boolean>();
```

Comment:
the name isLibPackageCache no long makes sense here. Just libPackageCache should be good